worktrolly-ui/src/app/icons/priority-icons/priority-icons.component.html

```diff
@@ -1,17 +1,17 @@
 <ng-container *ngIf="Priority == 'Low' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: yellow; padding: right 16px;" data-toggle="tooltip" data-placement="top" title="Low">
```

Comment:
Move css to .css file, use color code for color, and fix padding right. It's not a right way to use padding.